### DISCUSSION TITLE
How to package my extensions that created with jcb?

### DISCUSSION THREAD

**Discussion by saman222** [joomengine/Joomla-Component-Builder#917] [Q&A] [answered]
I want zip all my extension to one pkg. how to that with jcb? is there any other solution without use octojpack?
thank you

**Comment by Llewellynvdm** [comment, marked as answer]
This tool should be the ideal option.

https://git.vdm.dev/agerix/com_extensiondistributor

**Reply by saman222** [reply]
Thank you but there is no readme.txt . Is there any youtube tutorial link for that? or can you say how to connect to our jcb extensions?

**Reply by Llewellynvdm** [reply]
This application currently comes with no documentation or support. You will have to workout how it works on your own. Which should be easy enough ;)
